server/routes/connector_status.py

```
@@ -663,6 +666,22 @@ def _run_check(provider: str, token_owner_id: str) -> tuple:
+    try:
+        with db_pool.get_admin_connection() as conn:
+            with conn.cursor() as cursor:
+                cursor.execute(
+                    """SELECT COUNT(*) FROM user_manual_vms
+                       WHERE (user_id = %s OR org_id = %s)
+                         AND connection_verified = TRUE""",
+                    (user_id, org_id),
+                )
+                count = cursor.fetchone()[0]
+        return {"connected": count > 0}
+    except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)